bugzilla_dashboard/components_query.json

```diff
@@ -0,0 +1,102 @@
+{
+    "P1Defect": {
+        "label": "P1s defect",
+        "parameters": {
+            "f1": "creation_ts", "o1": "greaterthaneq", "v1": "-1y",
+            "priority": "P1",
+            "resolution": "---",
+            "bug_type": "defect"
+        }
+    },
+    "unassignedBetaBugs": {
+        "label": "Unassigned tracked beta bugs",
+        "parameters": {
+            "f1": "cf_tracking_firefox_beta",
+            "o1": "anyexact",
+            "v1": "+,blocking",
+            "f2": "cf_status_firefox_beta", "o2": "equals", "v2": "affected",
+            "f3": "assigned_to", "o3": "equals", "v3": "nobody@mozilla.org"
+        }
+    },
+    "nightlyNewBug": {
+        "label": "Nightly New Regression",
+        "hidden": "true",
+        "parameters": {
+            "keywords": "regression,",
+            "keywords_type": "allwords",
+            "resolution": "---",
+  
+            "f1": "cf_status_firefox_nightly", "o1": "equals", "v1": "affected",
+            "f2": "OP", "j2": "OR",
+            "f3": "cf_status_firefox_beta", "o3": "anyexact", "v3": "unaffected, ?, ---",
+            "f6": "CP",
+            "f7": "flagtypes.name", "o7": "notsubstring", "v7": "needinfo",
+            "f8": "cf_tracking_firefox_nightly", "o8": "notequals", "v8": "-",
+            "f9": "keywords", "o9": "notsubstring", "v9": "stalled"
+        }
+    },
+    "nightlyCarryOver": {
+        "label": "Nightly carry over",
+        "hidden": "true",
+        "parameters": {
+            "keywords": "regression,",
+            "keywords_type": "allwords",
+            "resolution": "---",
+            "f1": "cf_status_firefox_nightly", "o1": "equals", "v1": "affected",
+            "f2": "OP", "j2": "OR", "n2": "1",
```

Comment:
the same here but n2:1 means "apply NOT operator the parenthesis f2" so if you remove f2, j2 then n3: 1 should be ok.